lib/endon/helpers.ex

```diff
@@ -104,7 +104,7 @@ defmodule Endon.Helpers do
   def where(repo, module, conditions, opts) do
     module
     |> add_where(conditions)
-    |> add_opts(opts, [:limit, :order_by, :offset, :preload])
+    |> add_opts(opts, [:limit, :order_by, :offset, :preload, :lock])
```

Comment:
Should `:lock` be added to `find_by` as well?

Reply:
Good catch!  Added in 28a7529